ts/src/sourcemapper/LICENSE

```diff
@@ -0,0 +1,201 @@
+                                 Apache License
```

Comment:
I think this license and the whole sourcemapper directory shouldn't appear in this PR since it's already submitted?

Reply:
I forgot to remove the LICENSE and README after rebasing. 

sourcemapper.ts will show changes to reverse the mapping,